modules/themes/bloodyHellMate.js

```diff
@@ -0,0 +1,27 @@
+const version = "0";
+
+let style;
+
+let obj = {
+  onImport: async () => {
+    goosemodScope.showToast("onImport()");
```

Comment:
This isn't needed.

Reply:
Ack I was testing something, forgot about that sorry.

Reply:
No worries. Solving via edit myself now.